Source/Core/Check.js

```diff
@@ -0,0 +1,144 @@
+/*global define*/
+define([
+    './defaultValue',
+    './defined',
+    './DeveloperError',
+    './isArray'
+    ], function(
+        defaultValue,
+        defined,
+        DeveloperError,
+        isArray) {
+    'use strict';
+
+    /**
+     * Contains functions for checking that supplied arguments are of a specified type
+     * or meet specified conditions
+     * @private
+     *
+     * @exports Check
+     */
+    var Check = {};
```

Comment:
Ah, I probably even wrote the `Transform` one.  In practice, it might not matter for these cases so don't bother changing.